frontend/src/pages/RideDetails/RideDetails.css

```diff
@@ -7,7 +7,7 @@
     height: 250px;
     background-color: #b6b3bf;
 }
-.left {
```

Comment:
this css class conflicts with the .left defined globally by semantic, which messes up the styling on the login page. If you want to override semantic styles, define a custom class name and put "!important" after each css line.